todo/tests/testcontainers/shared_mongo.py

```
@@ -0,0 +1,10 @@
+from todo.tests.testcontainers.mongo_container import MongoReplicaSetContainer
+
+_mongo_container = None
+
+def get_shared_mongo_container():
+    global _mongo_container
+    if _mongo_container is None:
+        _mongo_container = MongoReplicaSetContainer()
+        _mongo_container.start()
+    return _mongo_container
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)